keeper/keeper.go

```diff
@@ -24,6 +24,7 @@ import (
 	"bytes"
 	"context"
 	"encoding/hex"
+	errorsmod "errors"
```

Comment:
this feels wrong, std lib imports shouldn't be given an alias. instead we should give an alias to whatever package we are importing that collides with the std lib

Reply:
Makes complete sense to me! Updated!